src/controller/config.js

```diff
@@ -81,72 +91,30 @@ bodyTagBtn.addEventListener('click', async (event) => {
 
 })
 
-newBodyPartBtn.addEventListener('click', () => {
-    bodyPartModal.show()
-})
-
-bodyPartBtn.addEventListener('click', async () => {
-    let txt = bodyPartTxt.value
+saveBodyPartBtn.addEventListener('click', async () => {
+    let txt = bodyPartTitleTxt.value
     if(txt){
         let bodyPart = {
             'name': txt,
             'img': myImage.src,
             'coordinates': coordinatesMap
         }
+
         if(bodyPartId == null){
             bodyPartId = await window.api.generateId()
         }
-        bodyPartMap[bodyPartId] = bodyPart
-        isBodyPartSaved = true
-        bodyPartId = null
-        coordinatesMap = {}
-        reloadBodyPart()
+        window.api.addOrEditBodyPartById(bodyPartId, checklistId, bodyPart)
```

Comment:
save the body part to the checklist and reload home page so it reflects also close the config page